docs/index.md

```diff
@@ -4,54 +4,31 @@ page_description: Introduces the Docker Registry
-is collection of images. Users interact with the registry by pushing images to
-or pulling images from the registry. The Docker Registry includes several
-optional features that you can configure according to your needs.
+A Registry is a storage and content delivery system, holding named Docker images, available in different tagged versions. For example, the image `distribution/registry`, with tags `2.0` and `latest`.
 
-![](images/registry.png)
+Users interact with a registry by using docker push and pull commands. For example, `docker pull myregistry.com/stevvooe/batman:voice`.
 
-The architecture supports a configurable storage backend. You can store images
-on  a file system or on a service such as Amazon S3 or Microsoft Azure. The
-default storage system is the local disk; this is suitable for development or
-some small deployments.
+Storage itself is delegated to drivers. The default storage driver is the local posix filesystem, which is suitable for development or small deployments. Additional cloud-based storage driver like S3 and Microsoft Azure are also supported. People looking into using other storage backends may do so by implementing their own driver following the [Storage API](storagedrivers.md).
 
-Securing access to images is a concern for even the simplest deployment. The
-registry service supports transport layer security (TLS) natively. You must
-configure it in your instance to make use of it. You can also use a proxy server
-such as Nginx and basic authentication to extend the security of a deployment.  
+Since securing access to your hosted images is paramount, the Registry does natively support TLS. You can also enforce basic authentication through a proxy like Nginx.
 
-The registry repository includes reference implementations for additional
-authentication and authorization support. Only very large or public registry
-deployments are expected to extend the registry in this way.
+The registry github repository includes reference implementations for additional authentication and authorization methods. Only very large or public registry deployments are expected to extend the registry in this way.
 
-Docker Registry architecture includes a robust notification system. This system
-sends webhook notifications in response to registry activity.  The registry also
-includes features for both logging and reporting as well. Reporting is useful
-for large installations that want to collect metrics. Currently, the feature
-supports both New Relic and Bugsnag.
+Finally, the Registry includes a robust [notification system](notifications.md), calling webhooks in response to activity, and both extensive logging and reporting. Reporting is mostly useful for large installations that want to collect metrics. Currently, New Relic and Bugsnag are supported.
 
 ## Getting help
 
-Docker Registry is an open source project and is under active development. If
-you need help, would like to contribute, or simply want to talk about the
```

Comment:
I wholeheartedly disagree with the benefits:

 * any decent text-editor will reflow *to your settings*: it's easier to read and better preserves the intent to *not* wrap at 80 chars, instead of breaking things for everyone else that doesn't have the same settings as you
 * the perceived benefit of making it easier to comment inline in github hides doc defects in the first place: if the text is too long, then the message is not clear enough, not well structured enough

Furthermore, enforcing 80-chars does introduce a lot of churn (by possibly triggering reformat of several unrelated lines), which is highly undesirable from a revision perspective.

To put it otherwise: this "rule" diverts resources and attention from what really matters (documentation content quality) toward pedantic and outdated churning.

Sorry for my feelings are so strong on this, but I really believe we should spend more energy fixing (a not-so-great) docker documentation overall, and less on devising infatuated guidelines :)

In the hope that does make sense and helps moving this debate forward.

Thanks for bringing this up @thaJeztah